libs/features/mas/commerce/libs/commerce.d.ts

```diff
@@ -38,20 +38,12 @@ declare global {
         ) => T &
             PlaceholderConstants & { new (): InstanceType<T> & Placeholder };
 
-        type getLocaleSettings = (
```

Comment:
I'd just remove this whole file if @yesil has no obj

Reply:
yeah, let's discuss this on monday but i would agree too

Reply:
I agree, but I would like to move them to jsdoc. By just removing we will lose valuable typing information.